cdk/app/lib/microservices.ts

```diff
@@ -40,8 +40,7 @@ export class Microservices extends cdk.Stack {
       nameDescription.add(new consul_ecs.ECSConsulMeshExtension({
         ...baseProps,
         serviceDiscoveryName: 'name',
-      }));
-      nameDescription.add(new ecs_extensions.AssignPublicIpExtension());
```

Comment:
When we are removing assignPublicIpExtension then how are services are reachable via internet as I suppose we are now using private subnets?

Reply:
the `greeter` app is still assigned with HttpLoadBalancer, which runs on public subnet and it will be the way for user to reach the app. the other app namely `name` and `greetings` is not accessible from the internet